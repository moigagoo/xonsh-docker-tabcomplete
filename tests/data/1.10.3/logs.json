{
        "arguments": [
                "-f", "--follow",
                "--help",
                "--since",
                "-t", "--timestamps",
                "--tail=all"
        ],
        "commands": []
}
