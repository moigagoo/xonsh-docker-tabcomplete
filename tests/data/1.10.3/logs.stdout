
Usage:	docker logs [OPTIONS] CONTAINER

Fetch the logs of a container

  -f, --follow        Follow log output
  --help              Print usage
  --since             Show logs since timestamp
  -t, --timestamps    Show timestamps
  --tail=all          Number of lines to show from the end of the logs
